app/containers/BalanceContainer/index.js

```diff
@@ -0,0 +1,78 @@
+/*
+ *
+ * BalanceContainer
+ *
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import selectBalanceContainer from './selectors';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import styles from './styles.css';
+import BalanceCell from 'components/BalanceCell';
+import $ from 'jquery'; 
+import BigNumber from 'bignumber.js';
```

Comment:
Copied from MyEtherWallet, thats how they do it. 
BigNumber is a class used throughout all bitcoin related projects to handle balances